packages/template-retail-react-app/app/components/store-locator-modal/index.jsx

```diff
@@ -16,52 +17,44 @@ import {
     useBreakpointValue
 } from '@salesforce/retail-react-app/app/components/shared/ui'
 import StoreLocatorContent from '@salesforce/retail-react-app/app/components/store-locator-modal/store-locator-content'
-import {useSearchStores} from '@salesforce/commerce-sdk-react'
+
+// Others
 import {
-    DEFAULT_STORE_LOCATOR_COUNTRY_CODE,
+    DEFAULT_STORE_LOCATOR_COUNTRY,
     DEFAULT_STORE_LOCATOR_POSTAL_CODE,
-    STORE_LOCATOR_DISTANCE
+    STORE_LOCATOR_NUM_STORES_PER_LOAD
 } from '@salesforce/retail-react-app/app/constants'
-import {noop} from '@salesforce/retail-react-app/app/utils/utils'
-import {useForm} from 'react-hook-form'
 
-const StoreLocatorModal = ({isOpen, onClose = noop}) => {
-    const intl = useIntl()
+export const StoreLocatorContext = createContext()
+export const useStoreLocator = () => {
+    const [userHasSetManualGeolocation, setUserHasSetManualGeolocation] = useState(false)
```

Comment:
I put useStoreLocator in this file to keep the folder surface area low and not in `/hooks` in anticipation for extensibility work (CC @bendvc)